### ISSUE TITLE
metadata and biosample ingest for NEON surface water metagenomes (DP1.20281.001)

### ISSUE THREAD

**Issue by aclum** [microbiomedata/issues#334]
**Deliverable this task is associated with**

_See Deliverables tab here: https://docs.google.com/spreadsheets/d/1jF1RU_TwQlJpqvHnKk-KieE6VlUv0je4eFddK9idPuE/edit?usp=sharing_

- 2

**RACI**

_Tag people in their roles_
- Responsible: @sujaypatil96 
- Accountable: @turbomam 
- Consulted: @aclum 
- Informed: @aclum 

**Describe the the task?**
- [ ] Ingest new NEON biosamples and metadata for surface water using [data release version RELEASE-2023](https://data.neonscience.org/data-products/DP1.20281.001/RELEASE-2023). nmdc study id is nmdc:sty-11-hht5sb92
Detailed plan for environmental context terms is defined in https://github.com/microbiomedata/issues/issues/273

**Criteria for completion**
- Records exist in prod mongo for surface water samples

**Estimate people time**

**Completion Date (Goal)**
-  Jan 19 

**Target Sprint Start & End Dates**
- Start: Jan 2
- End: Jan 26 

**Tag Blocker/Contingent upon issues**

TDB changes to ingest script make be required to support surface water samples

**Comment by ssarrafan** [comment]
This issue hasn't been updated in over a month. I'm removing from sprint and adding the backlog label.  @aclum @sujaypatil96

**Comment by ssarrafan** [comment]
@aclum is this going to be done by tomorrow?

**Comment by aclum** [comment]
No, moving to next sprint.

**Comment by aclum** [comment]
Removing from this sprint since Sujay is still working on the benthic samples so this won't be done by Friday.

**Comment by aclum** [comment]
Moving to the next sprint.

**Comment by ssarrafan** [comment]
Last day of sprint. Removing from sprint. Adding backlog label.

**Comment by aclum** [comment]
To be done in January.

**Comment by sujaypatil96** [comment]
Work on this issue is in progress on https://github.com/microbiomedata/nmdc-runtime/pull/430 but is currently blocked by incomplete ENVO mappings for the "stream" environmental context https://github.com/microbiomedata/issues/issues/273

**Comment by aclum** [comment]
Identified an issue today w/ the NEON to NMDC depth mapping, fixing this should make sure we have depth for all samples. Mark has a PR in with envo for the new stream biome terms.
The other minor update that is needed is to use the correct study id.

**Comment by ssarrafan** [comment]
I'll move to the next sprint per the last comments from Alicia and Sujay

**Comment by sujaypatil96** [comment]
This needs to be moved to the next sprint.

The PR that is tagged to this issue will be reviewed/merged after https://github.com/microbiomedata/nmdc-runtime/pull/448

**Comment by aclum** [comment]
There were 3 river/lake samples missing depth info, Hugh confirmed this manually to be 0.5 m. We were also directed to set depth for all surface water stream samples to .1 meters based on Hugh's conversation with the  NEON aquatic experts.

**Comment by ssarrafan** [comment]
@sujaypatil96 it looks like #448 has been merged.  Who needs to review this so we can close it out?

**Comment by aclum** [comment]
closing this issue, data is in prod. Getting the code merged in has a separate PR.
